leobot_control/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(leobot_control)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
```

Comment:
please also add install section to package https://github.com/AndriyPt/ros-navigation-examples/blob/kinetic-devel/moon_control/CMakeLists.txt